### ISSUE TITLE
Updates to Makerparty event resources and activities

### ISSUE THREAD

**Issue by lovegushwa** [MozillaFoundation/Advocacy#352]
Detailed description of work:
Small updates to https://learning.mozilla.org/en-US/events/resources before launch on the 20th.

_Start date: September 16, 2016_ 
_Due date: September 19, 2016_ 

CC: @damiankay @HPaulJohnson

**Comment by damiankay** [comment]
Reminder that we'll need to localize these in Tier 1 languages. Thx

**Comment by chrislarry33** [comment]
@damiankay what will be the process and deadline for getting them localized?

**Comment by HPaulJohnson** [comment]
Updates to Activities Section: 
- Remove the following activities:  Hack a business plan, Don't break the Internet, Startup Salon and Copyright Forum.
- Add two new activities: Post Crimes and Super Party

Final copy for final (4 activities):
1. "Post Crimes: Highlight the absurdity of outdated copyright laws through this fun activity that let’s you create slightly rebellious selfies and send them to your local lawmaker via postcard."
2. "Meme Around: Create memes that support copyright reform! You’ll learn how to use pictures from the public domain to gain a better understanding of the implications of outdated copyright law."
3. "Contributing to the Commons: Help people understand the value of content we can all use freely by contributing your own content under a creative commons license."
4. "Super Party: All three activities in one - Postcrimes, Meme School and Contributing to the Commons packaged together as a single activity."

cc: @ScottDowne

**Comment by damiankay** [comment]
@chrislarry33 - we can either leverage the L10n community or use our external agency. Given the fact that we have a week of testing, I'd suggest L10n, depending on when the activities are ready. 

I would think our deadline is Sep 23

**Comment by ldecoursy** [comment]
Thanks @HPaulJohnson - just updated the activity slide in the partner deck to reflect the language above.

**Comment by damiankay** [comment]
@HPaulJohnson - are the updates that you identified for the Maker Party pages on learning.moz? If so, I'd better create a ticket to add those strings for translation. 

@valianttry what's the process for asking L10n to translate copy updates after a page has been 100% translated and approved?

**Comment by damiankay** [comment]
@HPaulJohnson @chrislarry33 @ldecoursy can someone also point me to the actual activities? 

The original decision was to not translate these, but as per our group meeting the decision was made to translate them. The L10n community is having trouble keeping up with all of the translation requests, so I'm not sure if we can expect them to handle this additional work in time for public launch. 

CC @valianttry

**Comment by lovegushwa** [comment]
@ScottDowne all the updated event resources assets are here, including new thumbs: 
https://drive.google.com/drive/folders/0B2qu3ExKWIlvczM1REhkYVV4UU0?usp=sharing

Please update everything shown in the panel below:

<img width="1043" alt="screen shot 2016-09-18 at 2 14 16 pm" src="https://cloud.githubusercontent.com/assets/12228435/18619156/367e450a-7daa-11e6-8702-27f6bbbb86c9.png">

**Comment by lovegushwa** [comment]
@ScottDowne please update the "maker-party-logo.svg" file with this new file here:
https://drive.google.com/a/mozilla.com/file/d/0B2qu3ExKWIlvcUE1UENIcVNQOEk/view?usp=sharing

<img width="1127" alt="screen shot 2016-09-18 at 2 17 13 pm" src="https://cloud.githubusercontent.com/assets/12228435/18619179/b894192a-7daa-11e6-83bd-6be0c3250768.png">

**Comment by ScottDowne** [comment]
@HPaulJohnson or @lovegushwa where are the new thumbs for the new activities? Specifically, I need 1x and 2x thumbs for post crimes and super party. If these already exist, mind pointing me at them?

**Comment by ScottDowne** [comment]
@lovegushwa new maker party logo image is in eps format, which won't display in the browser. Mind converting it to an svg?

**Comment by ScottDowne** [comment]
@lovegushwa I'm not at all sure what you want me to do with the logos & assets part.

You're screenshot is the current one, and the google drive links seems to be random assets and not sure where or what you want done with them.

@damiankay I updated what I ccould, which is adding new copy for l10n (it's in a review, so first it needs to be merged into the repo, then it'll be on pontoon), but blocked on changing assets.

**Comment by damiankay** [comment]
@ScottDowne are you able to merge the copy into the repo, or does it require action from someone else?

**Comment by ScottDowne** [comment]
@damiankay it's waiting on review from Alan.

**Comment by damiankay** [comment]
@shaghdoosti are these through review yet? 

@chadsansing - do you have links to the new activities?

**Comment by chadsansing** [comment]
No - got a hold from @chrislarry33 who didn't seem sure these would be in the same format (in the curriculum template) given their new formatting. They only exist as docs at present.

**Comment by damiankay** [comment]
Ok, thanks for the update @chadsansing

**Comment by ScottDowne** [comment]
@damiankay I think then, we should figure out what we expect to do with the docs. I havn't seen them, but I'm guessing turning them into HTML like the rest would be ideal, but not sure if we have time at this point. I guess it depends on the complexity of the docs.

**Comment by damiankay** [comment]
@chadsansing could you please send us a link to do the docs so that we can review how to deploy them to the public?

**Comment by chadsansing** [comment]
Surely, though I believe 1.1.2. and 1.1.1. still need legal review, and 1.1.3. needs to be revised w/ legal's input.
- [1.1.1. Postcrimes](https://docs.google.com/document/d/1SYlvAUgnFXvcyyFiRxjkKv9w2d24FURbRM8oyQ03uhE/edit#heading=h.ucg424vhi68z)
- [1.1.2. Meme Around](https://docs.google.com/document/d/19KryggXsgF93vvmCYkNoFe7-OA60zMIOtKe4H47C7f8/edit#heading=h.fbejmkn7nsoj)
- [1.1.3. Share Your Photos with an Open License](https://docs.google.com/document/d/1nzndrPLLjSORnw1kVPyFE5n4geFFrtClwhoVl8y4uRU/edit#heading=h.t7nu9rr2rmv1)

I don't have a clear idea of when the content will be locked.

**Comment by damiankay** [comment]
@geroter @shaghdoosti we need to resolve these asap. 

Pls note that we need to allow time for localization as well. Thanks!

**Comment by damiankay** [comment]
@geroter just confirmed that he's meeting with Mika shortly to hopefully resolve! 🚀

**Comment by damiankay** [comment]
@geroter has confirmed that the activities are approved by Mika and are ready to ship. 

@chadsansing @ScottDowne 
1. Where should these live and how long will it take to set them up in their final location? 
2. How do we send these for localization? Will they go into Pontoon?

**Comment by chadsansing** [comment]
@damiankay 

1) I'm not sure where these should live. I think someone in more of a decider role should make the call; I would suggest a Mozilla repo. However, we also need to decide if these are also going to be in the curriculum template (like the last round), or if we are going to make them .pdfs (or something) with some campaign visual treatment since they have their own consistent format now.

2) I don't think the template is great for Pontoon, but I don't know. Would defer to @ScottDowne et al. here.

**Comment by damiankay** [comment]
Thanks @chadsansing 

@geroter @ldecoursy (can someone CC Lucy): Do you know whether these will be PDFs, built in the curriculum template, or another format? 

CC @HPaulJohnson @ScottDowne

**Comment by ldecoursy** [comment]
cc @lucyeoh

@chrislarry33 any decision about what format you'd like for these?

**Comment by chadsansing** [comment]
@ScottDowne: here are the updated activities; content should be cleared and locked as per @geroter's work with Mika & Raegan on the docs.

**Repo:** https://github.com/chadsansing/eu-maker-party-sample-activites

**Folder:** https://github.com/chadsansing/eu-maker-party-sample-activites/tree/master/updated-eu-maker-party-content

**Individual activities:**
- activity01-postcrimes.html - https://github.com/chadsansing/eu-maker-party-sample-activites/blob/master/updated-eu-maker-party-content/activity01-postcrimes.html
- activity02-meme-around.html - https://github.com/chadsansing/eu-maker-party-sample-activites/blob/master/updated-eu-maker-party-content/activity02-meme-around.html
- activity03-contribute-to-the-commons.html - https://github.com/chadsansing/eu-maker-party-sample-activites/blob/master/updated-eu-maker-party-content/activity03-contribute-to-the-commons.html
- combined-maker-party-activities.html - https://github.com/chadsansing/eu-maker-party-sample-activites/blob/master/updated-eu-maker-party-content/combined-maker-party-activities.html

cc @damiankay

**Comment by damiankay** [comment]
@ScottDowne can you please let me know as soon as these are in Pontoon?

**Comment by ScottDowne** [comment]
Staging link: https://learning.mofostaging.net/en-US/events/

Please use this to test.

**Comment by valianttry** [comment]
@ScottDowne when i click the activity links the main page just refreshes. What exactly are we testing?

**Comment by ScottDowne** [comment]
Everything but activity links. I'm still waiting for those.

**Comment by ScottDowne** [comment]
To be clear, I'm waiting for the activities in google doc form, not the actual translated webpages on say pontoon, those I'm still working on, which from my understanding would happen later.

**Comment by chadsansing** [comment]
@ScottDowne - the ones we have in hand (in my folder)? Those are all set, so far as I know, unless they need to be copied to another account. Here's the folder:

https://drive.google.com/drive/folders/0B0RBD2BW-srIdUV5d0NEVzNJY00?usp=sharing

Let me know when to change the sharing settings to whatever we need them to be.

**Comment by ScottDowne** [comment]
It's a localization issue. I'm not up to date on the why. But we decided to
use a Google doc or something while I worked to get them into pontoon
somehow.

**Comment by chadsansing** [comment]
Gotcha. Let me know if you need anything from me.

**Comment by ScottDowne** [comment]
Because I keep needing to dig up the live references, I'm dropping them here:

https://chadsansing.github.io/eu-maker-party-sample-activites/updated-eu-maker-party-content/activity01-postcrimes.html
https://chadsansing.github.io/eu-maker-party-sample-activites/updated-eu-maker-party-content/activity02-meme-around.html
https://chadsansing.github.io/eu-maker-party-sample-activites/updated-eu-maker-party-content/activity03-contribute-to-the-commons.html
https://chadsansing.github.io/eu-maker-party-sample-activites/updated-eu-maker-party-content/combined-maker-party-activities.html

**Comment by valianttry** [comment]
Localized activities were just delivered by Kontextor. Here is the zipped file @ScottDowne --

[delivery_translation_activities.zip](https://github.com/MozillaFoundation/Advocacy/files/503607/delivery_translation_activities.zip)

**Comment by ScottDowne** [comment]
If no one objects, I'll be moving these, and the host packs, into google docs.

**Comment by ldecoursy** [comment]
@KGorr is helping move the host packs into the Google slide template. So far she has:
- ES https://docs.google.com/a/mozillafoundation.org/presentation/d/12hDy1vnd73GBRXUxiFIN_sr7lJ8e_5k3vtQfYW5SfdI/edit?usp=xpa_n&ts=57ee91f2
- DE https://docs.google.com/a/mozillafoundation.org/presentation/d/1Or0fja5tUXGONvVEI9IiH772t2Rss1EoRDeCX32syzI/edit?usp=xpa_n&ts=57ee85a0

Others will be done by tomorrow.

**Comment by ScottDowne** [comment]
@ldecoursy Amazing, this is perfect! :rocket:

**Comment by ScottDowne** [comment]
@ldecoursy Possible to get the same treatment to the host pack pages/localizations?

**Comment by ldecoursy** [comment]
@ScottDowne Not sure what you mean. Those are the host packs. She's out next week though, so I think she'll only have time to get through the remaining few languages for this doc.

**Comment by ScottDowne** [comment]
@ldecoursy Ah, I see. This issue is for the activities, not the host packs. And moving them to google docs too, as they are currently in a zip file which isn't great.

**Comment by ScottDowne** [comment]
To be clear, both the localized host packs AND activities are in zip files, and I need to move them to google docs.

@ldecoursy So these are localized host packs? 
- ES https://docs.google.com/a/mozillafoundation.org/presentation/d/12hDy1vnd73GBRXUxiFIN_sr7lJ8e_5k3vtQfYW5SfdI/edit?usp=xpa_n&ts=57ee91f2
- DE https://docs.google.com/a/mozillafoundation.org/presentation/d/1Or0fja5tUXGONvVEI9IiH772t2Rss1EoRDeCX32syzI/edit?usp=xpa_n&ts=57ee85a0

**Comment by ldecoursy** [comment]
@ScottDowne Yes, that's the host pack. @kgorr is also working on moving the FR, IT, and PL versions of the host pack into Google Slides, but I don't think she'll have time to help with the activities.

**Comment by ScottDowne** [comment]
That's fine. I'm just glad one of them is being done. :)

I'll do the activities, likely.

**Comment by shaghdoosti** [comment]
Yep - and with the activities we want to get those up asap - but move them into the nicer html format as soon as possible. @ScottDowne does that make sense?

**Comment by ScottDowne** [comment]
@shaghdoosti Yup, I'm getting close to the html versions up and localized. Maybe 75% done?

One question I have is, the localizations in the zip, should map to the html pages? Thus the nicer html format content is already localized, and doesn't need to be on pontoon at all?

**Comment by ScottDowne** [comment]
Keeping this open for now, for the act of adding in the nicer HTML format or spinning that task into a new ticket, but for the simple localized content, it'll be done in this ticket: #498.

With that, I'm making #498 a P1 and this a P2.

**Comment by ScottDowne** [comment]
@valianttry @lovegushwa So, English is done. We can start QA on English. Here are the links.

https://advocacy-mozilla-org-staging.herokuapp.com/en-US/maker-party/activities/post-crimes
https://advocacy-mozilla-org-staging.herokuapp.com/en-US/maker-party/activities/meme-around
https://advocacy-mozilla-org-staging.herokuapp.com/en-US/maker-party/activities/contribute-to-the-commons
https://advocacy-mozilla-org-staging.herokuapp.com/en-US/maker-party/activities/combined-maker-party-activities

**Comment by ScottDowne** [comment]
Closing this, it's too confusing with #498. I'll just do all the work in #498 instead.
